### ISSUE TITLE
2 reviewers per PR

### ISSUE THREAD

**Issue by nsmith5** [sigstore/fulcio#374]
**Question**

Should we have 2 reviews per PR? I think this can be a great quality gate and as @haydentherapper mentioned it might be worth doing as we try to push the quality bar in Fulcio. If 2 reviews leads to slow reviews maybe we could add more maintainers to spread the burden?

**Comment by Foxboron** [comment]
It should probably be noted that 2 reviewers isn't only a quality assurance thing but also to prevent one account takeover from compromising the source.

**Comment by nsmith5** [comment]
Indeeeed ❤️ https://slsa.dev/spec/v0.1/requirements#two-person-reviewed 👀

**Comment by cpanato** [comment]
made a comment in the issue in the community repo

**Comment by dlorenc** [comment]
Left a comment over in the other issue - FWIW I feel the cons far outweigh the pros here.

The SLSA requirement is also a bit subtle and doesn't actually require two people to review every PR. The goal there is to ensure two different people have looked at it in some way, but the author can count as one of them. In practice this means that if one maintainer submits code, and another maintainer reviews that code, they count as two separate people as long as they are known to be two separate people :)

If an unknown contributor submits code, then two maintainers should look at it because we have no way of otherwise knowing that the unknown account is not a sock puppet of the original maintainer.

For reducing the risk of account compromise, we already require 2FA for every maintainer in this organization. I'd be in favor of requiring security tokens instead of SMS-based auth, but I don't think GitHub exposes that control yet.

In short, I think there are better ways of achieving the same goal here without the added friction and latency that a mandatory second reviewer provides.

**Comment by nsmith5** [comment]
Closing in favour of the community repo thread
